src/pages/dashboard/components/menu/menu.style.tsx

```diff
@@ -1,7 +1,53 @@
 import styled from '@emotion/styled'
 
 export const MenuStyled = styled('nav')`
+  display: flex;
+  flex-direction: column;
+  align-items: center;
   padding: 10px;
+  background: #ffffff;
+  width: 296px;
+`
+
+export const Logo = styled('div')`
+  margin-bottom: 60px;
+  display: flow;
```

Comment:
what is this ?
is this even a valid property value ?

ref:
https://developer.mozilla.org/en-US/docs/Web/CSS/display